spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -214,6 +216,8 @@
 
 	private KafkaExtendedBindingProperties extendedBindingProperties = new KafkaExtendedBindingProperties();
 
+	private Map<ConsumerDestination, ContainerProperties.AckMode> ackModeInfo = new HashMap<>();
```

Comment:
Probably safer to use a `ConcurrentHashMap<>()` we're not iterating so shouldn't be a problem but, just in case...